main.ts

```diff
@@ -41,8 +41,11 @@ class Translator {
     this.emit(')');
   }
 
-  visitFunctionLike(fn: ts.FunctionLikeDeclaration) {
-    this.visitParameters(fn);
+  visitFunctionLike(fn: ts.FunctionLikeDeclaration, accessor: string = null) {
+    if (fn.type) this.visit(fn.type);
+    if (accessor) this.emit(accessor);
+    if (fn.name) this.visit(fn.name);
+    if (accessor !== 'get') this.visitParameters(fn);
```

Comment:
Can you really not have parameters in a getter? Who enforces that? Maybe it's better to error out or just generate the code and have the Dart compiler error out?